README.md

```diff
@@ -4,6 +4,7 @@ A set of rake tasks for managing DNS records using [Terraform](https://terraform
 
 | Table of Contents |
 | ----------------- |
+| [Quick Start](#quick-start) |
 | [The Tasks](#the-tasks) |
```

Comment:
just noticed that the 'The Tasks' and 'Installation' are the wrong way round in the table of contents.